libraries/AP_HAL_ChibiOS/hwdef/JHEMCU-GF16-F405/hwdef.dat

```diff
@@ -0,0 +1,26 @@
+# hw definition file for processing by chibios_pins.py
+# JHEMCU GF16 by V-22
+# With F405 MCU, MPU6000 IMU and MAX7456 series OSD
+# Based on Mamba F405 hwdef from jeanphilippehell
+# thanks to betaflight for pin information (https://github.com/betaflight/unified-targets/blob/master/configs/default/JHEF-JHEF405PRO.config)
+# Board info https://github.com/jhemcu/FC-ESC-Firmware/tree/main/GF16-F405%26ESC
+
+include hwdef_pindef.dat
+
+define PRECISION_LANDING 0
+define OSD_ENABLED 1
+
+# --------------------- save flash ----------------------
+define HAL_BATTMON_SMBUS_ENABLE 0
```

Comment:
please use the defines in hwdef/include for these

Reply:
So i should add all the lines to save flash in hwdef/include/save_some_flash.inc ?

Reply:
found how to do it.